subjects/java/checkpoints/config-protector/README.md

```diff
@@ -0,0 +1,67 @@
+## Config Protector
+
+### Instructions
+
+Create a class `ConfigProtector` that provides a method to hide sensitive data in a configuration file using regex. The method should replace sensitive values with asterisks. The configuration file will be provided as a string, and the keys for the sensitive data will be given in a list.
+
+### Expected Class
+
+```java
+import java.util.List;
+
+public class ConfigProtector {
+    public String hideSensitiveData(String configFile, List<String> sensitiveKeys) {
+        // Implementation to hide sensitive data in the configuration file using regex
+    }
+}
+```
+
+### Usage
+
+Here is a possible `ExerciseRunner.java` to test your class:
+
+```java
+import java.util.Arrays;
+
```

Comment:
It's missing this import:
`import java.util.List;`

Reply:
fixed